sidebars.js

```diff
@@ -179,9 +179,7 @@ module.exports = {
         },
         'java/activities',
         'java/quick-start',
-        'java/workflow-interface',
-        'java/implementing-workflows',
-        'java/starting-workflow-executions',
+        'java/workflows',
```

Comment:
these are the java docs we collapsed into a single doc - need to doublecheck redirects before we merge this

Reply:
thanks!